pages/understanding-json-schema/_index.md

```diff
@@ -17,7 +17,7 @@ instructor for JSON Schema. It\'s for those that want to write it and
 understand it, but maybe aren\'t interested in building their own
 car—er, writing their own JSON Schema validator — just yet.
 
-> This tutorial describes JSON Schema draft 2020-12. Earlier versions of
+> This tutorial describes JSON Schema [draft](../learn/glossary#draft) 2020-12. Earlier versions of
```

Comment:
Just add a link to draft when draft is being used alone without pointing to one specific draft version. the version of this draft is "draft 2020-12" everything together. We'll link to draft when the word is used alone.

Reply:
Okay I will correct it.